modules/nixos/buildbot-repo-allowlist.nix

```diff
@@ -0,0 +1,23 @@
+[
+  "qowoz/microvm.nix"
+  "nix-community/authentik-nix"
```

Comment:
Isn't the github topic less work overall than having this list synchronously managed?